Neos.ContentGraph.DoctrineDbalAdapter/src/DoctrineDbalContentGraphProjection.php

```diff
@@ -519,7 +511,7 @@ private function getRelationPositionAfterRecalculation(
         usort(
             $hierarchyRelations,
             fn (HierarchyRelation $relationA, HierarchyRelation $relationB): int
-                => $relationA->position <=> $relationB->position
+            => $relationA->position <=> $relationB->position
```

Comment:
```suggestion
                => $relationA->position <=> $relationB->position
```

Doesn't matter all that much, but removing the indent here strikes me as unintentional.

Reply:
nice catch,. hmm oversaw that ^^ but as its not linted and _if_ basti had written the code originally it would probably be the same so id say oke ^^ have no strong preservations :D

Reply:
Yep, thx. That was my IDE doing funny things